test_caseinsensitivedict.py

```diff
@@ -100,12 +108,27 @@ def test_keys(self):
         # popitem()
         #     Remove and return a (key, value) pair from the dictionary.  Pairs
         #     are returned in LIFO order.
+    def test_popitem(self):
+        myinstance = CaseInsensitiveDict()
+        myinstance['Kaisa'] = 5
+        self.assertIn('Kaisa', myinstance)
+        item = myinstance.popitem()
+        self.assertEqual(item, ('Kaisa', 5))
+        self.assertNotIn(item, myinstance)
 
         # setdefault(key[, default])
         #     If key is in the dictionary, return its value. If not, insert key
         #     with a value of default and return default.  default defaults to
         #     None.
 
+    def test_setdefault(self):
+        myinstance = CaseInsensitiveDict()
+        myinstance['Kaisa'] = 5
+        item = myinstance.setdefault('Kaisa')
+        self.assertEqual(item, 5)
+        default_item = myinstance.setdefault('Jhin')
```

Comment:
`setdefault()` can take one or two parameters, but you're only testing the single parameter case